packages/realm-server/server.ts

```diff
@@ -44,7 +51,32 @@ import {
   MatrixBackendAuthentication,
   Utils,
 } from '@cardstack/runtime-common/matrix-backend-authentication';
-import jwt from 'jsonwebtoken';
+import {
+  TokenExpiredError,
+  JsonWebTokenError,
+  sign,
+  verify,
+} from 'jsonwebtoken';
+
+interface RealmServerTokenClaim {
+  user: string;
+}
+
+const DEFAULT_PERMISSIONS = Object.freeze([
+  'read',
+  'write',
+  'realm-owner',
+]) as RealmPermissions['user'];
+
+const IGNORE_SEED_FILES = [
+  'node_modules',
+  '.gitignore',
+  '.realm.json',
+  '.template-lintrc.js',
+  'package.json',
+  'TODO.md',
+  'tsconfig.json',
+];
```

Comment:
instead of hard coding this, I wonder if it would be more eligant to have a special ignore file in the seed realm that contains these entries. what do you think?

Reply:
I think this is fine